### PR TITLE
WRO-3085: [React18] Update dependencies of performance-tests

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

Updated enact, sandstone, react dependencies to latest versions

### CODE REVIEW

File: performanceMetrics/package.json [hunk 1]
```diff
@@ -30,16 +30,16 @@
...
-		"@enact/sandstone": "^2.1.2",
-		"@enact/spotlight": "^4.1.2",
-		"@enact/ui": "^4.1.2",
+		"@enact/core": "^4.5.0-alpha.1",
+		"@enact/i18n": "^4.5.0-alpha.1",
+		"@enact/sandstone": "^2.5.0-alpha.1",
+		"@enact/spotlight": "^4.5.0-alpha.1",
+		"@enact/ui": "^4.5.0-alpha.1",
 		"axios": "^0.24.0",
 		"classnames": "^2.2.5",
```
Comment (daniel-stoian-lgp) [inline performanceMetrics/package.json:39]:
```suggestion
		"classnames": "^2.3.1",
```

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: performance/TestResults.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/* eslint no-console: ["error", { allow: ["warn", "error"] }] */
+/* eslint-disable no-console, no-undef */
```
Comment (stanca-pop-lgp) [inline performance/TestResults.js:1]: Maybe you could leave one line empty below to keep the same style.

Comment (adrian-cocoara-lgp) [reply performance/TestResults.js:1]: Sure

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: package.json [hunk 3]
```diff
@@ -29,17 +29,17 @@
...
-		"@enact/spotlight": "^4.0.7",
-		"@enact/ui": "^4.0.7",
+		"@enact/core": "^4.5.0-alpha.1",
+		"@enact/i18n": "^4.5.0-alpha.1",
+		"@enact/sandstone": "^2.5.0-alpha.1",
+		"@enact/spotlight": "^4.5.0-alpha.1",
+		"@enact/ui": "^4.5.0-alpha.1",
 		"dotenv": "^6.2.0",
 		"node-fetch": "^2.3.0",
 		"prop-types": "^15.6.2",
```
Comment (MikyungKim) [inline package.json:39]: Shouldn't this be updated?

File: package.json [hunk 4]
```diff
@@ -29,17 +29,17 @@
...
 		"qs": "^6.6.0",
-		"react": "^17.0.1",
-		"react-dom": "^17.0.1",
+		"react": "^18.0.0",
+		"react-dom": "^18.0.0",
 		"react-router-dom": "^5.2.0",
 		"yargs": "^12.0.5"
 	},
```
Comment (MikyungKim) [inline package.json:45]: Does this repo use jest 24?

**Review by MikyungKim — COMMENTED** [review]
If some variables from global, you can do like
```
/* global page, minFPS, someGlobalVariables */
```
instead of disabling lint errors(/* eslint-disable no-undef */). Please fix them all.
Also, instead of using `/* eslint-disable react/jsx-no-bind */`,
Please fix them with guide from https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#react-hooks.

File: package.json (hunk 3 above)
Comment (adrian-cocoara-lgp) [reply package.json:39]: Right. I updated it to ^15.8.1

File: package.json (hunk 4 above)
Comment (adrian-cocoara-lgp) [reply package.json:45]: It uses "jest": "^24.1.0" on line 48.

Comment (MikyungKim) [reply package.json:45]: I meant, shouldn't it update to at lest 27 since we are using it.

Comment (adrian-cocoara-lgp) [reply package.json:45]: Oh, yes. I'll update it as well. Thanks!

**Pull request merged**